week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -21,9 +21,36 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 */
 //1)Create the function inside, Think about nested loops
 function sortAges(arr) {
-  //create function here
+  let arr1 = [];
+  for (let i = 0; i < arr.length; i++) {
+    if (Number.isInteger(arr[i])) {

```

Comment:
This will work perfectly for the test cases. However, what if you don't have an integer passed
another way to fix it is to check the type of the item